go.mod

```diff
@@ -33,7 +33,7 @@ require (
 	k8s.io/apimachinery v0.20.2
 	k8s.io/client-go v12.0.0+incompatible
 	knative.dev/eventing v0.20.1
-	knative.dev/eventing-kafka v0.20.1-0.20210202112232-900179eb4a86
+	knative.dev/eventing-kafka v0.0.0-00010101000000-000000000000
```

Comment:
I don't get this... tbh

Reply:
Not sure either. But this version probably doesn't matter. What matters is the version in the `replace` section below.